src/MacroExpander.js

```diff
@@ -155,6 +157,30 @@ export default class MacroExpander implements MacroContextInterface {
         }
     }
 
+    /**
+     * Skips ahead until finding the \else or \fi that ends the skipped text
+     * keeping track of \if...\fi nesting.
+     */
+    skipConditionalText() {
+        const condition = this.conditions.pop();
+        let depth = 1;
+        while (depth) {
+            const token = this.popToken();
+            if (conditionals.hasOwnProperty(token.text)) {
+                depth++;
+            } else if (token.text === "\\fi" ||
+                    (depth === 1 && !condition && token.text === "\\else")) {
+                depth--;
+                if (token.text === "\\else") {
+                    this.conditions.push(condition);
+                }
+            } else if (token.text === "EOF") {
+                throw new ParseError("Incomplete conditional! " +
+                    "End of input while skipping conditional text.", token);
```

Comment:
LaTeX says `Incomplete \iffalse; all text was ignored after line <n>.` or `Incomplete \iftrue; all text was ignored after line <n>.` (in the case of `\iftrue ... \else ...`).  Perhaps `` `Incomplete \\if${condition}` `` would be appropriate and more informative?